### PR TITLE
Add quiz-card animation

### PR DESCRIPTION
**Pull request by luziehtan**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -24,8 +24,8 @@ <h2 class="quiz-card__question">
           <img src="./bookmark_clear.svg" width="30" height="30" alt="" />
```
Comment (macfrei) [inline index.html:24]: As discussed with Jerry earlier, use a `button`-element instead of a `div` for the bookmark.

File: index.html [hunk 2]
```diff
@@ -24,8 +24,8 @@ <h2 class="quiz-card__question">
...
         </div>
         <details>
-          <summary>Show answer</summary>
-          <div class="quiz-card__answer">Tomorrow.</div>
+          <summary class="quiz-card__summary">Show answer</summary>
+          <div class="quiz-card__answer details">Tomorrow.</div>
```
Comment (macfrei) [inline index.html:28]: This could be a headline or paragraph instead of a div.

File: index.html [hunk 3]
```diff
@@ -24,8 +24,8 @@ <h2 class="quiz-card__question">
...
         </details>
         <div class="quiz-card__tags">
```
Comment (macfrei) [inline index.html:30]: You could also use a list instead of a div and style the list items as buttons.

**Review by macfrei — COMMENTED** [review]
Very nice animation! I added a few comments in your HTML. Keep up the good work! 🚀

File: index.html (hunk 1 above)
Comment (luziehtan) [reply index.html:24]: Changed as `span`-Element. Also in bookmark.html.

File: index.html (hunk 2 above)
Comment (luziehtan) [reply index.html:28]: Changed into `p`-Elements.

**Pull request merged**